src/libs/ComposerUtils/index.js

```diff
@@ -25,4 +25,18 @@ function canSkipTriggerHotkeys(isSmallScreenWidth, isKeyboardShown) {
     return (isSmallScreenWidth && DeviceCapabilities.canUseTouchScreen()) || isKeyboardShown;
 }
 
-export {getNumberOfLines, updateNumberOfLines, insertText, canSkipTriggerHotkeys};
+/**
+ * Find length of common ending of two strings
```

Comment:
```suggestion
 * Finds the position of the first difference between two strings
```

Reply:
Good point, let's do:

```suggestion
 * Finds the position of the first difference between two strings
 * This function starts the comparison from the end of the given strings
```

Reply:
I think it become a little bit confusing, `getFirstDifferencePosition` its name and description looks opposite of each other

Reply:
Ha, good point, the name and description do seem like they are opposites of each other. How about the function name `getCommonSuffixIndex`?

```suggestion
 * Returns the index of the first difference between two input strings,
 * starting from the end (suffix) of the strings.
 *
 * @param {String} str1
 * @param {String} str2
 * @returns {Number} The index of the first difference, indicating the end of the common suffix.
 */
 function getCommonSuffixIndex(str1, str2) {
```

Reply:
But we are not returning index of suffix, we are returning its length. We can modify it to return index, but two strings usually have different lengths and which one's index we return?

Reply:
Okay, how about:

```suggestion
 * Returns the length of the common suffix between two input strings.
 * The common suffix is the number of characters shared by both strings
 * at the end (suffix) until a mismatch is encountered.
 *
 * @param {string} str1
 * @param {string} str2
 * @returns {number} The length of the common suffix between the strings.
 */
function getCommonSuffixLength(str1, str2) {
```